package_installs.R

```diff
@@ -12,6 +12,7 @@ M <- 16
 options(Ncpus = parallel::detectCores())
 
 # Install parallel library.
+library(devtools)
```

Comment:
Should this be here? It at least probably shouldn't come between the comment about the parallel library

Reply:
Sorry, my last commit was missing.